src/users/entities/user.entity.ts

```
@@ -1,44 +1,52 @@
-import {
-  Column,
-  CreateDateColumn,
-  Entity,
-  PrimaryGeneratedColumn,
-  UpdateDateColumn,
-} from 'typeorm';
+import { Entity, Column, Unique, OneToMany } from 'typeorm';
+import { BaseEntity } from '@/common/entity/base.entity';
+import { Reservation } from '@/reservations/entities/reservation.entity';
+import { NotificationLog } from "@/notification-logs/entities/notification-log.entity";
```

ESLint

[error] Replace `"@/notification-logs/entities/notification-log.entity"` with `'@/notification-logs/entities/notification-log.entity'`

(prettier/prettier)